### PR TITLE
Add Keccak table to Table.md

### LINKED ISSUE #250: Define keccak table API
A small table that tells which columns and their values for a lookup to the keccak table

Transform this: https://github.com/privacy-scaling-explorations/zkevm-specs/blob/master/specs/keccak256/variable-length-input.md#lookup
Into something like this: https://github.com/privacy-scaling-explorations/zkevm-specs/blob/master/specs/tables.md

### PR DESCRIPTION
**Pull request by ChihChengLiang**
Fix #250

### CODE REVIEW

File: specs/tables.md [hunk 1]
```diff
@@ -202,6 +202,20 @@ From this table, the following columns contain values using the RLC encoding:
 - ValuePrev
 - ValueCur
 
+## Keccak Table
+
+See [tx.py](src/zkevm_specs/tx.py)
+
+| IsEnabled | InputRLC   | InputLen | Output      |
+| --------- | ---------- | -------- | ----------- |
+| 1         | $input_rlc | 200      | $output_rlc |
```
Comment (ed255) [inline specs/tables.md:211]:
```suggestion
| 1         | $input_rlc | $input_length | $output_rlc |
```

File: specs/tables.md [hunk 2]
```diff
@@ -202,6 +202,20 @@ From this table, the following columns contain values using the RLC encoding:
...
+
+Column names in circuit:
+- IsEnabled: `is_enabled`
```
Comment (ed255) [inline specs/tables.md:214]: Does the `is_enabled` column map to the `state_tag` column in the keccak circuit directly?  If so, does that mean that the tag value `Finalize` is `1`?

Comment (ChihChengLiang) [reply specs/tables.md:214]: The recently merged keccak has an `is_final` column, so it's a binary approach. https://github.com/privacy-scaling-explorations/zkevm-circuits/blob/d16ac9b8bda39c45c82a41d2d59054c85abbc9a1/zkevm-circuits/src/keccak_circuit/keccak_packed_multi.rs#L824

I'm thinking of nuking the current keccak spec and then adding a new spec to reflect the new implementation.

Comment (ed255) [reply specs/tables.md]: > The recently merged keccak has an `is_final` column, so it's a binary approach. https://github.com/privacy-scaling-explorations/zkevm-circuits/blob/d16ac9b8bda39c45c82a41d2d59054c85abbc9a1/zkevm-circuits/src/keccak_circuit/keccak_packed_multi.rs#L824

Thanks for the clarification!

> I'm thinking of nuking the current keccak spec and then adding a new spec to reflect the new implementation.

Sounds good!

**Review by ed255 — APPROVED** [review]
LGTM!

File: specs/tables.md [hunk 3]
```diff
@@ -202,6 +202,20 @@ From this table, the following columns contain values using the RLC encoding:
 - ValuePrev
 - ValueCur
 
+## Keccak Table
+
+See [tx.py](src/zkevm_specs/tx.py)
+
+| IsEnabled | InputRLC   | InputLen | Output      |
+| --------- | ---------- | -------- | ----------- |
+| 1         | $input_rlc | $input_length | $output_rlc |
```
Comment (CPerezz) [inline specs/tables.md:211]: What about this?
```suggestion
| bool         | $input_rlc | $input_length | $output_rlc |
```

**Review by CPerezz — APPROVED** [review]
LGTM feel free to ignore my comment.

File: specs/tables.md (hunk 3 above)
Comment (CPerezz) [reply specs/tables.md:211]: I mention this because this allows 0 to disable the lookup when needed. Linkin 0 -> 0

**Pull request merged**